src/commonMain/kotlin/arrow.exact/Exact.kt

```diff
@@ -1,23 +1,33 @@
 package arrow.exact
 
-import arrow.core.*
+import arrow.core.Either
 
-interface Exact<A, out B> {
 
-  fun from(value: A): Either<ExactError, B>
+interface Exact<A, out R : Refined<A>> : ExactEither<ExactError, A, R>
 
-  fun fromOrNull(value: A): B? {
-    return from(value).getOrNull()
-  }
+data class ExactError(val message: String)
```

Comment:
This is better a _not open_ 😁 ❤️

Reply:
Yeah... Idk, why I suggested `open` in the first place - it makes zero sense :rofl:  Maybe it's the fact that I'm doing Arrow Exact PR reviews at 2 am from my mobile phone xD